src/Foundation.Find.Cms/Config/SelectOneEnumAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using EPiServer.Shell.ObjectEditing;
+using System;
+using System.Web.Mvc;
+
+namespace Foundation.Find.Cms.Config
```

Comment:
Change namespace to Foundation.Find.Cms.Facets.Config

Reply:
Done